### ISSUE TITLE
Is there a way to populate the "about" dialog?

### ISSUE THREAD

**Issue by scottresnik** [diffplug/goomph#21]
I was curious if there was a way to populate the about dialog of a goomph eclipse install (Help -> About)?  Now it only shows `%productBlurb`.

**Comment by nedtwigg** [comment]
Not currently, but it could be added.  The branding stuff is here: https://github.com/diffplug/goomph/blob/master/src/main/java/com/diffplug/gradle/oomph/BrandingProductPlugin.java

**Comment by scottresnik** [comment]
I've got something working for this on my fork.  However, I'd like to pull the latest copy of the about.mappings file from from the p2 cached repository area.  For example, if I configure a standard Java developer eclipse distor, the about.mappings file is in `~/.goomph/shared-bundles/plugins/org.eclipse.epp.package.java_4.6.1.20161007-1200`.  Is there a good way to access that directory from the within a build, or within the plugin?

**Comment by nedtwigg** [comment]
I assume that the bundle in question is going to end up in the IDE that you're creating?  If so, the most automatic way is probably to use `SetupAction`, and look for the `about.mapping` file using a URL along the lines of `platform:/plugin/org.eclipse.epp.package.java/pathTo/about.mappings`.

My gut in this case is to lean more towards "manual" intervention - e.g. "if you want to populate the about dialog, find this jar, extract these files, copy them here, etc."  But if you're able to get the automatic working, more power to you :)

**Comment by scottresnik** [comment]
Your assumption is correct.  I went a pseudo manual route of copying the file with a simple copy task, in my project, after goomph setup the workspace but before it launches the ide.

**Comment by nedtwigg** [comment]
So PR #21 adds the basic functionality, and then you do something manual on top to get deeper customization?

**Comment by scottresnik** [comment]
Yes:

```groovy
task copyAboutMapping(type: Copy) {
  shouldRunAfter ideSetupWorkspace
  from(new File(com.diffplug.gradle.GoomphCacheLocations.bundlePool(), 'plugins')) {
    include 'org.eclipse.epp.package.java_*/about.mappings'
  }
  into new File(new File(eclipseDir, com.diffplug.gradle.FileMisc.macContentsEclipse()), "dropins/com.diffplug.goomph.branding")
  eachFile {FileCopyDetails fcp ->
    String[] segments = [fcp.name]
    fcp.path = fcp.name
    fcp.relativePath = new RelativePath(true, segments)
  }
}
ide.dependsOn copyAboutMapping
```

This allows me to have the actual version of the eclipse distro I'm running.  I couldn't really think of a clean way to apply it to the plugin, otherwise I would have put it in there.

**Comment by nedtwigg** [comment]
I don't see a good way to include it either :-/.  Glad you found a good workaround though :)

Eclipse 4.6.2 ships on 12/21.  If you're okay waiting until then, that'll be the next time I turn the crank to ship this in a release.

**Comment by scottresnik** [comment]
Waiting until 12/21 is fine with me.

**Comment by nedtwigg** [comment]
Published in `3.7.0`.
